docs-src/cloud/migration.md

```diff
@@ -0,0 +1,88 @@
+The volume of these requests might be quite high to execute against all the matches to a `ListFilter`.
+
+### Considerations when resuming Workflows on a new Cluster or Namespace
+
+- **Skipping Steps:** If your Workflow steps cannot guarantee idempotency, determine if you need to skip those steps when resuming the execution in the target Namespace.
+- **Elapsed Time:** If your Workflow is “resuming sleep” when in the target Namespace, determine how you will calculate the delta for the sleep invocation in the new execution.
+- **Child Relationships:** If your Workflow has Child Workflow relationships (other than Detached Parent Close Policy children), determine how you can pass the state of those children into the parent to execute the child in a resumed state.
+- **Heartbeat state:** If you have long running activities relying on heartbeat state, determine how you can resume these activities in the target Namespace.
+- If Child Workflows are the same type as their Parent types, those will be returned in List Filters being used to gather relevant executions. Unless these are Detached `ParentClosePolicy` children, this is not what you want since the Parent/Child relationship will not be carried over to the target Namespace.
+- Long running activities that use heartbeat details will not receive the latest details in the target Namespace.
+- Duration between Awaitables inside Workflow definition needs to be considered for elapsed time accuracy when resuming in the target Namespace.
+- Signaling directly from one Workflow to another needs to handle NotFound executions in the target Namespace since they may resume out of order.
+
+### Other considerations when migrating
+
+- Have you added an mTLS certificate to your Temporal Namespace? Review our [documentation for adding a certificate to your Temporal Cloud account](/cloud/certificates-intro) for more information.
+- There are differences in how metrics are generated in self-hosted Temporal and Temporal Cloud. Review the [documentation on Temporal Cloud metrics](/cloud/metrics-intro) for more information.
+- Consider the implications for [security and access to your cluster](/cloud/security-cloud-intro).
+- Review your current load (actions per second) and speak to your Account Executive and Solutions Architect so we can set appropriate Namespace [limits](/cloud/limits).
+
+### Multi-Cluster Replication
+
+[Multi-Cluster Replication](/self-hosted/what-is-multi-cluster-replication) is an experimental feature which asynchronously replicates Workflow Executions from active Clusters to other passive Clusters, for backup and state reconstruction.
+Migrating Execution History from a self-hosted Temporal Cluster to Temporal Cloud is not currently supported.
+However, a migration tool based on Multi-Cluster Replication which will enable this is currently in development for Temporal Cloud.
```

Comment:
I believe the clause "which will enable this" needs be surrounded by commas as a non-restrictive clause.